07-routing-gossip.md

```diff
@@ -450,7 +445,11 @@ The origin node:
   channel peer, even though the channel has not yet been announced (i.e. the
   `announce_channel` bit was not set).
     - MUST set the `short_channel_id` to either an `alias` it has
-	  received from the peer, or the real channel `short_channel_id`.
+    received from the peer, or the real channel `short_channel_id`.
+    - if it sets `short_channel_id` to an `alias`:
+      - MUST set `scid_alias` to 1 in `message_flags`
+    - otherwise:
+      - MUST set `scid_alias` to 0 in `message_flags`
```

Comment:
What about older nodes?

Reply:
They already set it to `0`, otherwise they wouldn't be spec-compliant, because of this requirement (that has been here forever): `MUST set bits in channel_flags and message_flags that are not assigned a meaning to 0.`

Reply:
Ah ok, missed that they were bits